tensorflow/python/tools/import_pb_to_tensorboard.py

```diff
@@ -65,7 +66,8 @@ def import_to_tensorboard(model_dir, log_dir):
 
 
 def main(unused_args):
-  import_to_tensorboard(FLAGS.model_dir, FLAGS.log_dir)
+  _ = unused_args
```

Comment:
To improve pylint score

Reply:
Something seems wrong with the pylintrc because "unused_" is meant to be recognized as a dummy argument. Prefer to use "def main(_)" as a workaround to avoid the warning rather than reassigning the variable.

Reply:
I was unsure about changing an arg named by someone else, but, sure, I can do that.